README.md

```diff
@@ -50,55 +56,63 @@ Table of Contents
 
 ```bash
 $ oc apply -f https://raw.githubusercontent.com/kiegroup/kogito-cloud/master/s2i/kogito-imagestream.yaml -n openshift
 ```
 
-The installation on OpenShift 4.x is pretty straightforward since Kogito Operator is available in the OperatorHub as a community operator. 
+### Deploy to OpenShift 4.x
 
-Just follow the OpenShift Web Console instructions in the _Catalog_, _OperatorHub_ section in the left menu to install it in any namespace in the cluster. 
+Kogito operator is a namespaced operator, which means that you will need to install it into the namespace you want your kogito application to run in.
 
-![Kogito Operator in the Catalog](docs/img/catalog-kogito-operator.png?raw=true)
+#### Via OperatorHub automatically
 
-You can also [run the operator locally](#running-locally) if you have the [requirements](#requirements) configured in your local machine.
+The installation on OpenShift 4.x is pretty straightforward since Kogito Operator is available in the OperatorHub as a community operator. The Operator can be easily found by filtering by _Kogito_ name.
 
-### Deploy to OpenShift 4.x Manually
+Just follow the OpenShift Web Console instructions in the _Catalog_, _OperatorHub_ section in the left menu to install it in any namespace in the cluster. 
 
-Make sure that the Kogito image stream is created in the cluster:
+![Kogito Operator in the Catalog](docs/img/catalog-kogito-operator.png?raw=true)
 
-```bash
-$ oc apply -f https://raw.githubusercontent.com/kiegroup/kogito-cloud/master/s2i/kogito-imagestream.yaml -n openshift
-```
+#### Via OperatorHub manually
 
-Then create an entry in the OperatorHub catalog:
+If it does not appear, you can still install it manually, by creating an entry in the OperatorHub Catalog:
 
 ```bash
 $ oc create -f deploy/olm-catalog/kogito-cloud-operator/kogitocloud-operatorsource.yaml
 ```
 
 It will take a few minutes for the operator to become visible under the _OperatorHub_ section of the OpenShift console _Catalog_. The Operator can be easily found by filtering by _Kogito_ name.
 
+Then, you can install as decribed in [the automatic process](#via-operatorhub-automatically).
+
+#### Via local Operator
+
+Or you can also [run the operator locally](#running-locally) if you have the [requirements](#requirements) configured in your local machine.
+
+#### Check Operator is running
+
 Verify operator availability by running:
 
 ```bash
-$ oc describe operatorsource.operators.coreos.com/kogitocloud-operator -n openshift-marketplace
+$ oc describe operatorsource.operators.coreos.com/kogitocloud-operator -n {YOUR_NAMESPACE}
 ```
 
 ### Deploy to OpenShift 3.11
 
 Installation on OpenShift 3.11 has to be done manually since the OperatorHub catalog is not available by default:
 
 ```bash
-## kogito imagestreams should already be installed/available ... e.g.
-$ oc apply -f https://raw.githubusercontent.com/kiegroup/kogito-cloud/master/s2i/kogito-imagestream.yaml -n openshift
```

Comment:
We might need to have this since users just scan the doc and go directly into this section, skipping prereq. Having the is created again won't cause a harm